04-spatial-operations.Rmd

```diff
@@ -676,10 +676,13 @@ if (knitr::is_latex_output()){
 
 Imagine that we need to join the `capacity` variable in `cycle_hire_osm` onto the official 'target' data contained in `cycle_hire`.
 This is when a non-overlapping join is needed.
-The simplest method is to use the topological operator `st_is_within_distance()`, as demonstrated below using a threshold distance of 20 m (note that this works with projected and unprojected data).
+The simplest method is to use the topological operator `st_is_within_distance()`, as demonstrated below using a threshold distance of 20 m.
+Note that this works with projected data and also with unprojected data.
+In the latter case, however, one needs to make sure that the spherical geometry engine (s2) is switched on which it is by default in **sf** (see section \@ref(s2)).
```

Comment:
:+1:

Reply:
Shorter version could be:

```
For unprojected (e.g. lon/lat CRSs such as WGS84) the spherical geometry engine (s2) must be enabled for `st_within_distance()` to work,, as it is in **sf** (see section \@ref(s2)).
```

Reply:
Yes, way better, let's use your version. I will update as soon as I am back on my laptop, for now, I need to get some sleep. Happy new year by the way 😊

Reply:
Happy New Year :fireworks:

Reply:
I slightly rewrote: 

```
One can set the threshold distance in metric units also for unprojected data (e.g. lon/lat CRSs such as WGS84), if the spherical geometry engine (s2) is enabled, as it is in **sf** by default (see section \@ref(s2)).
```